frame/democracy/src/lib.rs

```diff
@@ -848,6 +930,29 @@ decl_module! {
 			<NextExternal<T>>::kill();
 		}
 
+		/// Remove a proposal.
+		///
+		/// The dispatch origin of this call must be _Root_.
+		///
+		/// - `ref_index`: The index of the proposal to cancel.
+		///
+		/// # <weight>
+		/// - Complexity: `O(1)`.
+		/// - Db writes: `ReferendumInfoOf`
+		/// - Base Weight: 21.57 µs
+		/// # </weight>
+		#[weight = T::WeightInfo::cancel_proposal()]
+		fn cancel_proposal(origin, #[compact] prop_index: PropIndex) {
```

Comment:
ummm --  didn't we usually add new dispatchables at the bottom to prevent breaking the call indices?

Reply:
+1